core_impl_android/README.md

```diff
@@ -0,0 +1,6 @@
+Instrumentation Android implementation
+======================================================
+
+* Java 6 and Android compatible.
```

Comment:
This will be only android compatible. Once android move forward with supporting more java7 we will move this to java7. So I would say just "Android compatible".